### PR TITLE
Update Arch Linux(AUR) version

### PR DESCRIPTION
**Pull request by zayn7lie**
I have created a AUR version for such a perfect open-source project: <https://aur.archlinux.org/packages/neothesia>

It is my hope that the project could get more and more popular.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,9 +16,12 @@ If you have any questions, feel free to join my Discord
 [![Video](https://i.imgur.com/1R5uOnA.png)](https://youtu.be/ReE9nVuMCSE)
 
 ## Download
+
 <a href="https://flathub.org/apps/details/com.github.polymeilex.neothesia"><img width="240" alt="Download on Flathub" src="https://flathub.org/assets/badges/flathub-badge-en.png"/></a>
 
-Or binary releases:
+Arch Linux(AUR): <https://aur.archlinux.org/packages/neothesia>
```
Comment (PolyMeilex) [inline README.md:22]: Don't take this personally, but could we mention that the code of the package is not audited by Neothesia, for example `(Unofficial, maintained by @zayn7lie)` or something along those lines.

Feel free to word it however you like. I just want to indicate to people that perhaps they should read the script before installing.

**Comment by PolyMeilex** [comment]
Thanks, this is a great improvement!

File: README.md (hunk 1 above)
Comment (zayn7lie) [reply README.md:22]: Sure (Sorry, I forget about this~)

**Comment by PolyMeilex** [comment]
AUR Version bump to 0.2 would be welcome, should be a trivial tag change, I don't think any deps changed. Thanks in advance!

**Comment by zayn7lie** [comment]
> @zayn7lie AUR Version bump to 0.2 would be welcome, should be a trivial tag change, I don't think any deps changed. Thanks in advance!

Updated. Thanks for notification.

**Comment by PolyMeilex** [comment]
@zayn7lie
AUR Version bump to 0.3.1 would be welcome.  Thanks in advance.

**Pull request merged**